DynamicActiveRecord.php

```diff
@@ -157,7 +160,7 @@ public static function dynColDecode($encoded)
         // control characters U+0000 through U+001F. This causes JSON decoders to fail.
         // This workaround escapes those characters.
         $encoded = preg_replace_callback(
-            '{[\x00-\x1f]}',
+            '/[\x00-\x1f]/',
```

Comment:
`{...}` is for the *advanced* regex user - it never conflicts. `/.../` conflicts too often.